src/konnektor/__init__.py

```diff
@@ -2,33 +2,27 @@
 # For details, see https://github.com/OpenFreeEnergy/konnektor
 
 # basic gufe types:
-from gufe import Component, SmallMoleculeComponent, ProteinComponent
+from gufe import Component, ProteinComponent, SmallMoleculeComponent
 
 # Konnektor content
 from .network_planners import (
-    MaximalNetworkGenerator,
+    ClusteredNetworkGenerator,
+    CyclicNetworkGenerator,
     HeuristicMaximalNetworkGenerator,
-    RadialLigandNetworkPlanner,
-    NNodeEdgesNetworkGenerator,
+    MaximalNetworkGenerator,
     MinimalSpanningTreeNetworkGenerator,
-    CyclicNetworkGenerator,
-    ClusteredNetworkGenerator,
-    StarrySkyNetworkGenerator,
     MstConcatenator,
-)
-
-from .network_tools import (
-    concatenate_networks,
-    merge_networks,
-    append_component,
-    delete_transformation,
+    NNodeEdgesNetworkGenerator,
+    RadialLigandNetworkPlanner,
+    StarrySkyNetworkGenerator,
 )
 from .network_tools import (
     ChargeClusterer,
-    ScaffoldClusterer,
     ComponentsDiversityClusterer,
+    ScaffoldClusterer,
+    append_component,
+    concatenate_networks,
+    delete_transformation,
+    merge_networks,
 )
-
-from . import network_analysis
```

Comment:
I deleted this because, if moved to the top, it causes circular imports. I don't think it's actually needed, but please correct me.